aiida/engine/processes/workchains/workchain.py

```diff
@@ -289,9 +289,17 @@ def _update_process_status(self) -> None:
         """Set the process status with a message accounting the current sub processes that we are waiting for."""
         if self._awaitables:
             status = f"Waiting for child processes: {', '.join([str(_.pk) for _ in self._awaitables])}"
-            self.node.set_process_status(status)
         else:
-            self.node.set_process_status(None)
+            status = None
+        if self.paused:
+            # update the pre-paused status so that when the process is played
+            #  it will be set to the correct status
+            self._pre_paused_status = status  # pylint: disable=attribute-defined-outside-init
+        else:
+            self.node.set_process_status(status)
+            # we also need to update the process status
+            # because it is used to initialize the _pre_paused_status
+            self.set_status(status)
```

Comment:
This is double up, because `Process.set_status` calls `self.node.set_process_status(status)`.